core/trino-main/src/main/java/io/trino/server/DynamicFilterService.java

```diff
@@ -309,6 +313,16 @@ public boolean isAwaitable()
         };
     }
 
+    public void registerDynamicFilterConsumer(QueryId queryId, DynamicFilterId dynamicFilterId, Consumer<Domain> domainConsumer)
```

Comment:
Make it accept `Set<DynamicFilterId> dynamicFilters`

`domainConsumer` -> `consumer`